src/main/java/org/spdx/licenseTemplate/LicenseTextHelper.java

```diff
@@ -34,7 +34,7 @@ public class LicenseTextHelper {
 	// most of these are comments for common programming languages (C style, Java, Ruby, Python)
 	protected static final Set<String> SKIPPABLE_TOKENS = Collections.unmodifiableSet(new HashSet<>(
 		Arrays.asList("//","/*","*/","/**","#","##","*","**","\"\"\"","/","=begin","=end")));
-	static final String DASHES_REGEX = "[\\u2012\\u2013\\u2014\\u2015]";
+	static final String DASHES_REGEX = "[\\u2012\\u2013\\u2014\\u2015\\-]{1,2}";
```

Comment:
```suggestion
	static final String DASHES_REGEX = "[\\u2010\\u2011\\u2012\\u2013\\u2014\\u2015\\uFE58\\uFF0D\\-]{1,2}";
```

Just in case if you would like to have these hyphens/dahses as well:

- Hyphen (\u2010)
- Non-Breaking Hyphen (\u2011)
- Small Em Dash (\uFE58)
- Fullwidth Hyphen-Minus (\uFF0D)